USAGE.md

```diff
@@ -0,0 +1,105 @@
+
+With this skill you can control items that respond to on/off and numeric commands.
+The Skill as well has limited support for thermostats.
+The skill is supported for English (U.S.), English (U.K.), and German languages. 
+
+## General Configuration Instructions
+
+### Requirements
+
+* [openHAB Cloud Connector](http://docs.openhab.org/addons/io/openhabcloud/readme.html) configured using myopenHAB.org
+* Amazon account
+* Echo, Echo Dot, or another way to use Alexa (e.g. [Reverb](https://reverb.ai/))
+<!--- did I miss something? --->
+
+### 1. Item Configuration
+
+You have to tag your items to make them useable by Alexa.
+See [Tagging](http://docs.openhab.org/configuration/items.html#tagging) for more details.
+
+* **Items via .items - File**
+
+  Some examples of tagged items are:
+  
+  ```java
+  Switch Kitchen_Light "Kitchen Light" <light> (gKitchen) [ "Lighting" ]
+  Dimmer Bedroom_Light "Bedroom Light" <light> (gBedroom) [ "Lighting" ]
+  Number Bedroom_Temperature "Bedroom Temperature" (gBedroom) [ "CurrentTemperature" ]
+  
+  Group gDownstairsThermostat "Downstairs Thermostat" (gFF) [ "Thermostat" ]
+  Number Downstairs_Thermostat_CurrentTemp "Downstairs Thermostat Current Temperature" (gDownstairsThermostat) [ "CurrentTemperature" ]
+  Number Downstairs_Thermostat_Target_Temperature "Downstairs Thermostat Target Temperature" (gDownstairsThermostat) [ "TargetTemperature" ]
+  String Downstairs_Thermostat_Heating_Cooling_Mode "Downstairs Thermostat Heating/Cooling Mode" (gDownstairsThermostat) [ "homekit:HeatingCooling
+  ```
+  
+  Please make sure to place your tag infront of the channel-informations. e.g.
+  See [Item Definition and Syntax](http://docs.openhab.org/configuration/items.html#item-definition-and-syntax)
+  
+   ```java
+   Switch Kitchen_Light "Kitchen Light" <light> (gKitchen) [ "Lighting" ] {channel="..."}
+   ```
+   
+* **Items via PaperUI**
+ 
+   There (actually) is no easy way to tag yout Items via PaperUI.
+ 
+   Some users managed to tag items via REST API.
+   Have a loot at [this post](https://community.openhab.org/t/apply-tags-to-items-added-linked-in-paper-ui/19443/11?u=mboremski).
+   <!--- Are there more relevant ways to configure items? --->
+   <!--- Should we add a chapter for availabletags? --->
+
+### 2. Skill Configuration
+
+1. Visit the [Alexa-Website](https://alexa.amazon.com/) or use the belonging app on your mobile.
+2. Navigate to "Smart Home" -> "Configure Smart Home" and search for openHAB.
+3. Follow the Instructions there to authorise Alexa using your myopenhab.org-account.
+
+### 3. Final Configuration
+
+After tagging your items you can go back to the Alexa-configurations and search for devices.
+
+### Example Voice Commands
+
+Here are some example voice commands:
+
+ * "Alexa. turn on Kitchen Light"
+ * "Alexa, turn off Kitchen Light"
+ * "Alexa, turn on Bedroom Light"
+ * "Alexa, turn on Bedroom Light"
+ * "Alexa, dim Kitchen Lights to 30 percent"
```

Comment:
You've forgotten about the heating examples 😮

Reply:
Nope. I left it out willingly. :)
Tell me what to add an I'll do.